components/org.wso2.micro.integrator.ndatasource.core/src/main/java/org/wso2/micro/integrator/ndatasource/core/utils/DataSourceUtils.java

```diff
@@ -244,6 +251,46 @@ public static void secureResolveDocument(Document doc, boolean checkSecureVault)
 		}
     }
 
+	public static void secureResolveOMElement(OMElement doc, boolean checkSecureVault)
+			throws DataSourceException {
+
```

Comment:
It seems like the boolean checkSecureVault has no usage inside this method.

Reply:
Fixed